src/Confluent.Kafka/Library.cs

```diff
@@ -125,5 +126,17 @@ public static bool Load(string path)
         /// </summary>
         public static int HandleCount
             => kafkaHandleCreateCount - kafkaHandleDestroyCount;
+
+        internal static List<KeyValuePair<string, string>> NameAndVersionConfig
+        {
+            get
+            {
+                return new List<KeyValuePair<string, string>>
+                {
+                    new KeyValuePair<string, string>( "client.software.name", "confluent-kafka-dotnet"),
+                    new KeyValuePair<string, string>( "client.software.version", VersionString )
```

Comment:
note: The version string should be without prefixing v

Reply:
this is as reported by librdkafka, which looks to always be without a 'v'.